docs/en_US/install/cemu.md

```diff
@@ -1,36 +1,42 @@
-<div class="tip green">This Guide may be missing some info or incomplete.</div>
+<div class="tip green">This guide may be missing some info or incomplete.</div>
 
 # Cemu
 
 ## Download
 <div class="tip red">
 	<strong>Note:</strong>
-	Only experimental builds of Cemu 2.0 are supported. At this time Cemu does not have a stable release of Cemu 2.0 which supports Pretendo
+	Only experimental builds of Cemu 2.0 are supported. At this time, Cemu does not have a stable release of Cemu 2.0 which supports Pretendo.
 </div>
 
-Cemu 2.0 has official built-in support for Pretendo as of October 10, 2022. Head to the Cemu GitHub [releases](https://github.com/cemu-project/Cemu/releases) page and download the latest Cemu experimental release (tagged as `Pre-release`). Only `Cemu 2.0-5 (Experimental)` and above is supported at the moment. Additionally you may build Cemu from source using the provided [build instructions](https://github.com/cemu-project/Cemu/blob/main/BUILD.md)
+Cemu 2.0 introduced official built-in support for Pretendo on October 10th, 2022. Head to the Cemu GitHub [releases](https://github.com/cemu-project/Cemu/releases) page and download the latest Cemu experimental release (tagged as `Pre-release`). Only `Cemu 2.0-5 (Experimental)` and above is supported at the moment. Alternatively, you may build Cemu from source using the provided [build instructions](https://github.com/cemu-project/Cemu/blob/main/BUILD.md).
```

Comment:
I think it should be bumed up to 2.0-16 as there were some pretendo bugs I fixed (mainly content-type not being set correctly buy)